core/src/test/java/org/bitcoinj/core/PeerTest.java

```diff
@@ -970,4 +973,20 @@ public void bitcoinSerializeToStream(OutputStream stream) throws IOException {
                     || (e instanceof SocketException && e.getMessage().equals("Socket is closed")));
         }
     }
+
```

Comment:
This could go into its own SendHeadersMessageTest class, to be consistent with the other *MessageTest classes.